perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/client/ExceptionResolverImpl.java

```diff
@@ -120,7 +119,7 @@ private void resolve() {
 			}
 
 			setInfo(trans.alreadySubmitted(getBeanName()),
-					text + trans.visitSubmitted(Window.Location.getHref().split("#")[0], application.getId(), trans.submittedTitle()));
+					text + trans.visitSubmitted(Window.Location.getHref().split("#")[0], application.getId(), trans.openSubmittedTitle()));
```

Comment:
Label `openSubmittedTitle()` doesn't exist now, but I think that `myApplications()` would be sufficient here since the same label is also presented in the header menu.
```suggestion
					text + trans.visitSubmitted(Window.Location.getHref().split("#")[0], application.getId(), trans.myApplications()));
```